client.go

```diff
@@ -38,6 +37,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/tozny/e3db-clients-go/searchExecutorClient"
+
```

Comment:
I think this should go after line 46 perhaps, but if your IDE says differently I won't argue

Reply:
The saddest little dependency all the IDEs moving it around